octane/evmengine/keeper/abci.go

```diff
@@ -117,9 +117,18 @@ func (k *Keeper) PrepareProposal(ctx sdk.Context, req *abci.RequestPreparePropos
 	}
 
 	// Create execution payload message
-	payloadData, err := json.Marshal(payloadResp.ExecutionPayload)
-	if err != nil {
-		return nil, errors.Wrap(err, "encode")
+	var payloadData []byte
```

Comment:
Just a suggestion:
```
	var (
		payloadData  []byte
		payloadProto *types.ExecutionPayloadDeneb
	)
```

Reply:
if it was 3 variables, sure, but with only 2, your solution is 4 lines of code, while mine is only 2 #lessismore